Roles/Impostor/Amateur.cs

```diff
@@ -0,0 +1,211 @@
+﻿using UnityEngine;
+using static TOHE.Options;
+
+namespace TOHE.Roles.Impostor;
+
+internal class Amateur : RoleBase
+{
+    private static float RealTimeKillCooldown = 0f;
+    private static PlayerControl SelfTarget = null;
+    private static bool IsRevealed = false;
+    private static bool IsInMeeting = false;
+    private static bool SetUpForAirship = false;
+    //===========================SETUP================================\\
+    private const int Id = 28700;
+    private static readonly HashSet<byte> playerIdList = [];
+    public static bool HasEnabled => playerIdList.Any();
+    public override bool IsEnable => HasEnabled;
+    public override CustomRoles ThisRoleBase => CustomRoles.Impostor;
+
+    public override Custom_RoleType ThisRoleType => Custom_RoleType.ImpostorKilling;
+    //==================================================================\\
+
+    private static OptionItem KillCooldown;
+    public static OptionItem IncreaseKillCooldownOnVent;
+    public static OptionItem IncreaseKillCooldownAmount;
+    public static OptionItem SetKillCooldownOffDistance;
+    private System.IntPtr NativeMethodInfoPtr_RpcShapeshift_Public_Void_PlayerControl_Boolean_0;

```

Comment:
lol what is this

Reply:
Wdym?

Reply:
I didn't even notice that lol, I don't know why that's there I never even used that variable.